bin/cli-helpers.js

```diff
@@ -7,9 +7,24 @@ function loadVerifyExamplesConfig(configPath) {
   */
 
   var verifyExamples = null;
+  var elmJson = null;
 
   try {
     verifyExamples = require(configPath);
+    if (verifyExamples["elm-root"] === undefined) {
+      verifyExamples["elm-root"] = "..";
+    }
+    if (verifyExamples["root"] !== undefined) {
+      throw new Error(
+        `elm-verify-examples.json at ${configPath} has a root key, but it should be elm-root and point to the root of the Elm project`
+      );
+    }
+    var elmJsonPath = path.join(
+      path.dirname(configPath),
+      verifyExamples["elm-root"],
+      "elm.json"
+    );
+    elmJson = require(elmJsonPath);
   } catch (e) {
     console.log(`Copying initial elm-verify-examples.json to ${configPath}`);
     fsExtra.copySync(
```

Comment:
So perhaps here we can now just remove this bit, and instead just 

```js
var elmJsonPath = path.join(
       path.dirname(configPath),
       "..",
       "elm.json"
     );
     elmJson = require(elmJsonPath);
   verifyExamples = {};
}
```

or some such

Reply:
oh I thought we just make it optional not remove it completely, but maybe removing it is fine as well.

Reply:
By optional I mean that the file doesn't have to exist - if it doesn't you get sensible defaults that should work for most projects.

Reply:
I think we're slightly talk past each other. What I've been proposing is:

*Make the existence of `elm-verify-examples.json` file optional. If it is missing, the default place we find elmRoot is in the containing folder. The default for which modules to run is the exposed ones plus Readme.md.*

This change instead *removes the ability to configure the `elmRoot`*. I think that's also interesting, although probably to maintain some flexibility, I'd suggest searching containing folders recursively.